src/renderer/pathTracer.cpp

```diff
@@ -60,20 +67,23 @@ glm::vec3 Renderer::colorPixel(const unsigned int i, const unsigned int j,
   return colorRay(ray, scene);
 }
 
-glm::vec3 Renderer::colorRay(const Ray &ray, Scene &scene) {
+glm::vec3 PathTracer::colorRay(const Ray &ray, Scene &scene) {
   Hit hit = scene.getRayIntersection(ray);
   return hit.color();
 }
 
-Renderer::Renderer(unsigned int width, unsigned int height)
+PathTracer::PathTracer(unsigned int width, unsigned int height)
     : mWidth(width), mHeight(height),
       mAspectRatio(static_cast<float>(width) / static_cast<float>(width)) {}
```

Comment:
since we are making changes in this file I think this line is incorrect https://github.com/therenderingguys/path-tracer/issues/32